apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/PooledSession.cs

```diff
@@ -137,6 +134,33 @@ internal static PooledSession FromSessionName(SessionPool.ISessionPool pool, Ses
         /// </summary>
         internal DateTime RefreshTimeForTest => new DateTime(RefreshTicks, DateTimeKind.Utc);
 
+        /// <summary>
+        /// Finalizer which logs a warning for unreleased instances.
+        /// This does not automatically release the instance back to the pool, as that would
```

Comment:
nit: release the session (not the instance)?

Reply:
Not sure which is best to be honest, but will change :)